R/AOO_functions.R

```diff
@@ -199,12 +198,25 @@ getMinAOO <- function(ecosystem.data, grid.size,
                  min.percent.rule = min.percent.rule,
```

Comment:
in the previous line: grid = shift(grid, x = x[1], y = x[2]), doesn't x[2] just make the value default to NA? 
Since optim() only takes 1 variable for optimization, doesn't that mean it just searches diagonally (x=y)? Or am I completely misunderstanding what x[2] does...